docker/build.mk

```diff
@@ -22,11 +22,13 @@ BASE_TAG ?= gcr.io/fuzzbench
 build-all: $(addsuffix -all, $(addprefix build-,$(FUZZERS)))
 pull-all: $(addsuffix -all, $(addprefix pull-,$(FUZZERS)))
 
+# If we're running using GitHub Workflows, cache-from a remote image. Otherwise
+# just use the local cache.
+cache_from = $(if ${GITHUB_RUN_ID},--cache-from $(1),)
 
 base-image:
 	docker build \
-    --tag $(BASE_TAG)/base-image \
-    --cache-from $(BASE_TAG)/base-image \
+    --tag $(BASE_TAG)/base-image $(call cache_from,${BASE_TAG}/base-image) \
```

Comment:
better to split into new line for easier reading.

Reply:
Fixed!